CMakeLists.txt

```diff
@@ -163,6 +163,7 @@ ngraph_var(NGRAPH_USE_PREBUILT_LLVM DEFAULT "FALSE")
 option(NGRAPH_UNIT_TEST_ENABLE "Control the building of unit tests" TRUE)
 option(NGRAPH_TOOLS_ENABLE "Control the building of tool" TRUE)
 option(NGRAPH_CPU_ENABLE "Control the building of the CPU backend" TRUE)
+option(NGRAPH_USE_MKLDNN_V1 "Use MKLDNN-v1.0" FALSE)
```

Comment:
v1 will be the default at some point. shouldn't we have the opposite option like "NGRAPH_USE_LEGACY_MKLDNN" and set "TRUE" as default and phase it out at some point?

Reply:
will change the option. thanks.